pages/spicedb/modeling/validation-testing-debugging.mdx

```diff
@@ -85,6 +88,13 @@ project:docs#admin:
   - "[user:rauchg] is <platform:vercel#admin>"
 ```
 
+## Check Tracing
+
+SpiceDB supports tracing of check requests to view the path(s) taken to compute the result, as well as timing information.
+
+This information is requested via a header (versions older than v1.31.0) or setting `with_tracing: true` (v1.31.0 or later), and
```

Comment:
Might be more clear to separate the info here into separate sections: Versions previous to v1.31.0 and versions v1.31.0 on

Also specify where to set with_tracing eg in the request body.